Games/Niko: Through The Dream/Steam/script.json

```diff
@@ -0,0 +1,7 @@
+{
+	"scriptName": "Steam",
+	"compatibleOperatingSystems": ["MACOSX", "LINUX"],
+	"testingOperatingSystems": ["LINUX"],
```

Comment:
What is requiresNoCD? (see [script.json](https://github.com/PlayOnLinux/Scripts/wiki/script.json))